extensions/3DTILES_multiple_contents/README.md

```diff
@@ -0,0 +1,393 @@
+          "uri": "subtrees/{level}/{x}/{y}.subtree"
+        }
+      }
+    }
+  }
+}
+```
+
+Example subtree JSON:
+
+```jsonc
+{
+  "buffers": [
+    {
+      "byteLength": 262160
+    }
+  ],
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteLength": 43691,
+      "byteOffset": 0
+    },
+    {
+      "buffer": 0,
+      "byteLength": 131072,
+      "byteOffset": 43696
+    },
+    {
+      "buffer": 0,
+      "byteLength": 43691,
+      "byteOffset": 174768
+    },
+    {
+      "buffer": 0,
+      "byteLength": 43691,
+      "byteOffset": 218464
+    }
+  ],
+  "tileAvailability": {
+    "bufferView": 0
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 1
+  },
+  "extensions": {
+    "3DTILES_multiple_contents": {
+      "contentAvailability": [
+        {
+          "bufferView": 2
+        },
+        {
+          "bufferView": 3
+        }
+      ]
+    }
+  }
+}
+```
+
+<!-- omit in toc -->
+#### Metadata Groups in Implicit Tiling
+
+If both the [`3DTILES_implicit_tiling`](../3DTILES_implicit_tiling) and [`3DTILES_metadata`](../3DTILES_metadata) extensions are used, each content template URI can be assigned to a metadata group.
+
+Example tileset JSON:
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "layer": {
+            "properties": {
+              "color": {
+                "type": "ARRAY",
+                "componentType": "UINT8",
+                "componentCount": 3
+              },
+              "order": {
+                "componentType": "INT32"
+              }
+            }
+          }
+        }
+      },
+      "groups": {
+        "buildings": {
+          "class": "layer",
+          "properties": {
+            "color": [128, 128, 128],
+            "order": 0
+          }
+        },
+        "trees": {
+          "class": "layer",
+          "properties": {
+            "color": [10, 240, 30],
+            "order": 1
+          }
+        }
+      }
+    }
+  },
+  "root": {
+    "refine": "ADD",
+    "geometricError": 16384.0,
+    "boundingVolume": {
+      "region": [-1.707, 0.543, -1.706, 0.544, 203.895, 253.113]
+    },
+    "extensions": {
+      "3DTILES_multiple_contents": {
+        "content": [
+          {
+            "uri": "buildings/{level}/{x}/{y}.b3dm",
+            "extensions": {
+              "3DTILES_metadata": {
+                "group": "buildings"
+              }
+            }
+          },
+          {
+            "uri": "trees/{level}/{x}/{y}.i3dm",
+            "extensions": {
+              "3DTILES_metadata": {
+                "group": "trees"
+              }
+            }
+          }
+        ]    
+      },
+      "3DTILES_implicit_tiling": {
+        "subdivisionScheme": "QUADTREE",
+        "subtreeLevels": 10,
+        "maximumLevel": 16,
+        "subtrees": {
+          "uri": "subtrees/{level}/{x}/{y}.subtree"
+        }
+      }
+    }
+  }
+}
+```
+
+## Extension Schema Reference
```

Comment:
For consistency with the other specs, remove the generated schema reference and just link to the schema files